src/main/app/src/app/policy/model/zaak-acties.ts

```diff
@@ -9,7 +9,7 @@ export class ZaakActies {
     verlengen: boolean;
     hervatten: boolean;
     afbreken: boolean;
-    afsluiten: boolean;
+    voortzetten: boolean;
```

Comment:
voortzetten, moet dit niet afhandelen of behandelen , voortzetten klinkt als startenTaak of wijzigenZaak.